### PR TITLE
O3-3816 Add tab for rejected orders and prevent further actions on De…

### PR DESCRIPTION
**Pull request by CynthiaKamau**
…clined Tests

## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Add tab for rejected orders and prevent further actions on declined Tests

## Screenshots

https://github.com/user-attachments/assets/6a74d5de-eb09-4955-99ba-1180c49c538e

### CODE REVIEW

**Comment by ojwanganto** [comment]
Thanks, @CynthiaKamau. It would be worthy to show the reason for declining the order

File: src/index.ts [hunk 1]
```diff
@@ -52,6 +52,11 @@ export const completedLabRequestsTable = getAsyncLifecycle(
   options,
 );
 
+export const declinedLabRequestsTable = getAsyncLifecycle(
+  () => import('./lab-tabs/data-table-extensions/declined-lab-requests-table-extension'),
+  options,
+);
```
Comment (donaldkibet) [inline src/index.ts:58]:
```suggestion
import DeclinedLabRequstTable from './lab-tabs/data-table-extensions/declined-lab-requests-table-extension';
export const declinedLabRequestsTable = getSyncLifecycle(DeclinedLabRequstTable,options);
```

**Review by donaldkibet — APPROVED** [review]
Thanks @CynthiaKamau

File: src/index.ts (hunk 1 above)
Comment (CynthiaKamau) [reply src/index.ts:58]: I added this in line with how the other tabs have been exported

**Comment by CynthiaKamau** [comment]
> Thanks, @CynthiaKamau. It would be worthy to show the reason for declining the order

I have added it as part of the expansion slot

**Review by jabahum — APPROVED** [review]

**Review by pirupius — APPROVED** [review]

**Pull request merged**